docusaurus.config.js

```diff
@@ -107,16 +107,16 @@ const config = {
         darkTheme: darkCodeTheme,
       },
       typesense: {
-        typesenseCollectionName: 'camino-docs',
+        typesenseCollectionName: 'camino-docs-default',
         typesenseServerConfig: {
           nodes: [
             {
-              host: 'docs.camino.foundation',
-              port: 8108,
+              host: 'typesense.camino.network',
+              port: 443,
               protocol: 'https',
             },
           ],
-          apiKey: 'K3s7dltmYhPw0oqOepeOx8liLZPRAGH2',
+          apiKey: 'TfCsn1wHrrpAzkHQP7747iPAS83Rsqn3',
```

Comment:
shouldn't we use github secret here ?

Reply:
No. This is the read only search key. It's in the client-side javascript code already. So no need to hide it here.

Reply:
By the way, the admin api key used in the workflows is coming from github secrets as that key is only for scraping (writing to typesense)